inginious/frontend/templates/task.html

```diff
@@ -1,4 +1,4 @@
-$def with (user_info, course, task, submissions, students, eval_submission, user_task, previous_taskid, next_taskid, webterm_link, input_random_list=[])
+$def with (user_info, course, task, submissions, students, eval_submission, user_task, previous_taskid, next_taskid, webterm_link, input_random_list=[], adaptive_next_taskid=None)
```

Comment:
Your code should remain a plugin. Adding a parameter inside task.html related to your plugin strongly couple the INGInious code with your plugin, and that's not good.